pkg/bot/slack.go

```diff
@@ -85,7 +86,7 @@ func (b *SlackBot) Start() {
 		switch ev := msg.Data.(type) {
 		case *slack.ConnectedEvent:
 			log.Info("BotKube connected to Slack!")
-
+			setupDefaultChannelForKubeCtl(RTM.Client, b.ChannelName, b.ClusterName, b.DefaultNamespace)
```

Comment:
Hey @sgandon,
In case you have missed it, we don't have to specify `--cluster-name` flag if the command is executed from the configured channel.
In the case of multicluster setup, e.g `Cluster A -> Channel X` and `Cluster B -> Channel Y`, the BotKube commands posted in `Channel X` will be executed on `Cluster A`.
We also follow the same pattern for other integrations - Mattermost and Discord. Please confirm this doesn't change the existing behavior.